scavenger-schema/src/main/resources/db/changelog/test-data-set.sql

```diff
@@ -100,3 +100,222 @@ VALUES (1, 1, 'com.example.demo.additional.AdditionalService.get()', 0, 1,
         'METHOD'),
        (29, 2, 'com.example.demo.MyController', 2, 1, 'com.example.demo', 1, 1646027014967, 'CLASS'),
        (30, 2, 'com.example.demo', 2, 3, '', 1, 1646027014967, 'PACKAGE');
+
+
+--changeset scavenger:4
+
+TRUNCATE TABLE codebase_fingerprints;
```

Comment:
Is it possible to add without truncating?

Reply:
I think it's possible.
Even though it's still like that...
The test code needs more changes than I thought, so I'll roll back and work on a different PR.